### DISCUSSION TITLE
Text to Neo4j Cypher scripts

### DISCUSSION THREAD

**Discussion by nickzren** [microsoft/autogen#4019] [Show and tell]
Developed a lightweight AI agent-based framework that converts natural language queries into Cypher scripts for Neo4j, leveraging LLMs for precise query generation:
https://github.com/nickzren/text-to-cypher

Hope to have a discussion with someone who shares the same interests in this feature.

**Comment by vebsun** [comment]
This looks awesome. Let me try it out and share my feedback

**Comment by tyler-suard-parker** [comment]
I would be interested in a Neo4j agent.

**Comment by tyler-suard-parker** [comment]
I tried your script and it did not work for me, I keep getting syntax errors.
